stable/coredns/templates/configmap-autoscaler.yaml

```diff
@@ -15,9 +15,13 @@ metadata:
     kubernetes.io/name: "CoreDNS"
     {{- end }}
     app.kubernetes.io/name: {{ template "coredns.name" . }}-autoscaler
-{{- if .Values.customLabels }}
-{{ toYaml .Values.customLabels | indent 4 }}
-{{- end }}
+    {{- if .Values.customLabels }}
+    {{- toYaml .Values.customLabels | nindent 4 }}
+    {{- end }}
+  {{- if .Values.autoscaler.configmap.annotations }}
+  annotations:
+    {{- toYaml .Values.autoscaler.configmap.annotations | nindent 4 }}
```

Comment:
Do you need annotations only for the configmap or should be treat that similar as "customLabels" and apply it to every object?

Reply:
I think just for the configmap because CI/CD tend to treat them differently. I can add `autoscaler.customAnnotations` as well to all autoscaler objects?